kotlin.UninitializedPropertyAccessException in io.treehouses.remote.bases.BaseTerminalFragment.setBind

https://play.google.com/console/u/0/developers/4706131988394147634/app/4973925947152385508/vitals/crashes/60405b11/details?installedFrom=PLAY_STORE&days=30
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performDestroyActivity (ActivityThread.java:4076)
  at android.app.ActivityThread.handleDestroyActivity (ActivityThread.java:4094)
  at android.app.ActivityThread.handleRelaunchActivity (ActivityThread.java:4304)
  at android.app.ActivityThread.-wrap15 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1505)
  at android.os.Handler.dispatchMessage (Handler.java:111)
  at android.os.Looper.loop (Looper.java:207)
  at android.app.ActivityThread.main (ActivityThread.java:5765)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:789)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:679)
Caused by: kotlin.UninitializedPropertyAccessException: 
  at io.treehouses.remote.bases.BaseTerminalFragment.setBind (BaseTerminalFragment.java:46)
  at io.treehouses.remote.bases.BaseTerminalFragment.getBind (BaseTerminalFragment.java:46)
  at io.treehouses.remote.fragments.TerminalFragment.onDestroy (TerminalFragment.java:98)
  at androidx.fragment.app.Fragment.performDestroy (Fragment.java:3190)
  at androidx.fragment.app.FragmentStateManager.destroy (FragmentStateManager.java:758)
  at androidx.fragment.app.FragmentStateManager.moveToExpectedState (FragmentStateManager.java:367)
  at androidx.fragment.app.FragmentStore.moveToExpectedState (FragmentStore.java:120)
  at androidx.fragment.app.FragmentManager.moveToState (FragmentManager.java:1632)
  at androidx.fragment.app.FragmentManager.dispatchStateChange (FragmentManager.java:3109)
  at androidx.fragment.app.FragmentManager.dispatchDestroy (FragmentManager.java:3088)
  at androidx.fragment.app.FragmentController.dispatchDestroy (FragmentController.java:334)
  at androidx.fragment.app.FragmentActivity.onDestroy (FragmentActivity.java:322)
  at androidx.appcompat.app.AppCompatActivity.onDestroy (AppCompatActivity.java:242)
  at android.app.Activity.performDestroy (Activity.java:6479)
  at android.app.Instrumentation.callActivityOnDestroy (Instrumentation.java:1149)
  at android.app.ActivityThread.performDestroyActivity (ActivityThread.java:4063)
  at android.app.ActivityThread.handleDestroyActivity (ActivityThread.java:4094)
  at android.app.ActivityThread.handleRelaunchActivity (ActivityThread.java:4304)
  at android.app.ActivityThread.-wrap15 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1505)
  at android.os.Handler.dispatchMessage (Handler.java:111)
  at android.os.Looper.loop (Looper.java:207)
  at android.app.ActivityThread.main (ActivityThread.java:5765)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:789)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:679)
```

--- a/app/src/main/kotlin/io/treehouses/remote/fragments/TerminalFragment.kt
+++ b/app/src/main/kotlin/io/treehouses/remote/fragments/TerminalFragment.kt
@@ -91,13 +91,13 @@ class TerminalFragment : BaseTerminalFragment() {
     }
 
     override fun onDestroy() {
-        super.onDestroy()
         if(chatOpen()) {
             if (mChatService.state == Constants.STATE_NONE) {
                 mChatService.start()
                 updatePingStatus(bind.pingStatus, bind.PING, getString(R.string.bStatusIdle), Color.YELLOW)
             }
         }
+        super.onDestroy()
     }
 
     override fun onResume() {

